kubernetes/database.libsonnet

```diff
@@ -5,13 +5,13 @@ local resources = import 'resources.libsonnet';
   DatabaseCredential(name, app, namespace): k._Object('databases.outreach.io/v1', 'DatabaseCredential', name, app=app, namespace=namespace) {
     username:: error 'username is required',
     vault:: null,
-    iamauth:: false,
+    auth:: 'mysql',
```

Comment:
I am not sure I understand this ... DCO is not going to be supported in the future and we are going to use DPO for all of this. Can you explain why this is needed?

Reply:
I see the ctx here https://github.com/getoutreach/database-credential-operator/pull/73#discussion_r973388348 .. got it..